metadata.yml

```diff
@@ -0,0 +1,19 @@
+rules:
+- apiGroups: [""]
+  apiVersions: ["v1"]
+  resources: ["*"]
+  operations: ["CREATE", "UPDATE"]
+mutating: false
+contextAware: false
+executionMode: wasi
+annotations:
+  io.kubewarden.policy.title: annotation-validator
+  io.artifacthub.displayName: Annotation validator
+  io.artifacthub.resources: Any
+  io.artifacthub.keywords: compliance
+  io.kubewarden.policy.description: A policy that validates and mutates annotations
+  io.kubewarden.policy.author: "Kubewarden developers <cncf-kubewarden-maintainers@lists.cncf.io>"
+  io.kubewarden.policy.ociUrl: ghcr.io/kubewarden/policies/go-wasi-policy-template
+  io.kubewarden.policy.url: https://github.com/kubewarden/go-wasi-policy-template
+  io.kubewarden.policy.source: https://github.com/kubewarden/go-wasi-policy-template
```

Comment:
Diverge from the `oci-target` used in the release workflow

Reply:
thanks, fixed